scripts/downscoping-with-cab-setup.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
When/by whom or what is this script run? 

Doing this in bash is hacky. Not a total blocker for purely test code, but Java code would be preferable.

Reply:
This is a convenience script that makes it easy for us to port the integration tests to a new project or quickly set everything up if someone goes into pantheon and makes changes. It is similar to the script we had for the Workload Identity Federation integration tests. We decided to use a script as doing it in the test code means we may have to deal with potential intermittent test failures and recovery. You can take a look at go/cab-client-integration-testing if you want to read about our approach. 

It has already been run, so you don't need to run it. If you want to change the configuration, you can run it again. Ideally, this script will never even need to be executed again.